### PR TITLE
Set "tweet" appearance for short campaign updates.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This pull request updates the __Campaign Update__ block to display any content shorter than 144 characters as a "tweet", with no title and larger font size. It also pulls author information from the "additional content" blob on the custom block, and sets some defaults for optional fields.

screen shot 2017-02-28 at 11 06 42 am

### CODE REVIEW

File: resources/assets/components/CampaignUpdateBlock/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import classnames from 'classnames';
+import { markdown } from '../../helpers';
+import Block from '../Block';
+import Figure from '../Figure';
+import DEFAULT_AVATAR from './default-avatar.png';
+import './campaign-update.scss';
+
+const Byline = ({author, jobTitle = 'DoSomething.org Staff', avatar = DEFAULT_AVATAR}) => (
+  <Figure image={avatar}>
+    <strong>{author}</strong><br/>
```
Comment (weerd) [inline resources/assets/components/CampaignUpdateBlock/index.js]: `<br/>` 😱  :trollface:

File: resources/assets/components/CampaignUpdateBlock/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <p className="footnote">{jobTitle}</p>
+  </Figure>
+);
+
+const CampaignUpdateBlock = (props) => {
+  const { title, content, additionalContent } = props.fields;
```
Comment (weerd) [inline resources/assets/components/CampaignUpdateBlock/index.js]: do all the rest of the props get assigned to the `additionalContent` var? That's kinda cool :)

File: resources/assets/components/CampaignUpdateBlock/index.js (hunk 1 above)
Comment (DFurnes) [inline resources/assets/components/CampaignUpdateBlock/index.js]: Awww yeeee! 😈

File: resources/assets/components/CampaignUpdateBlock/index.js (hunk 2 above)
Comment (DFurnes) [inline resources/assets/components/CampaignUpdateBlock/index.js]: The `additionalContent` field is the JSON text area on the "Custom Block" content type, so it's already a separate collection of nested fields!

Comment (weerd) [inline resources/assets/components/CampaignUpdateBlock/index.js]: Gotcha, cool!

**Pull request merged**